### DISCUSSION TITLE
Small Break 🦙

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#361] [Announcements]
Hi everyone, sorry for the silence, a lot of stuff has happen in my personal life lately.

Long story short, I will have a small surgery tomorrow, it might make me unable to be on the PC for a while, to be safe I will take a small 20 days break to recover. There's another project that I've been working on and I hope to release it by the end of the year, I hope you guys look forward to it.

Depending on how much energy I have I might be able to read all the issues and pull requests by next week, but no promises.

To be clear, my life is not at risk, I'll be fine.

I will be attending classes virtually and playing with my switch for a while hahaha

Take care of yourselves!

![image](https://github.com/user-attachments/assets/25ebbd92-4106-4b2d-a3f8-e5787b029d35)

(Btw I will respond to this discussion if I survive the surgery :3)

**Comment by luyu-wu** [comment]
Hope you have a fast recovery!

**Comment by hwittenborn** [comment]
Best of luck man! I know we all need that break sometimes, so use it well 🙌🙌

**Comment by spaciouskarter78** [comment]
Get well soon

**Comment by Jeffser** [comment]
Update: I'm not dead :)

**Comment by heEXDe** [comment]
Video games can be a form of ... meditation. :lotus_position:
